dev/com.ibm.ws.webcontainer.servlet.3.1_fat/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/CDIUpgradeHandlerTest.java

```diff
@@ -458,22 +431,42 @@ private void closeSocket(Socket socket, String urlText) throws Exception {
 
     private static final String NULL_SESSION_ID = null;
 
-    private void displayLog(String sessionId, String[] expectedLog, String[] unexpectedLog) throws Exception {
-        WebBrowser logBrowser = createWebBrowser();
-        logBrowser.setAutoRedirect(false);
-        logBrowser.setAcceptCookies(true);
+    private void displayLog(String[] expectedLog) throws Exception {
```

Comment:
Minor comment -- I don't like the display logs method name. Makes it more confusing since it verifies the output for the test. 


Maybe call it verifyOutputMatchesExpectedLog or something similar?